db/seed.js

```diff
@@ -30,21 +31,34 @@ db.exec(sql, function (err) {
 // // Seed the database with some data
 console.log("Seeding database...");
 
-// Data to seed the database with
+// User data to seed the database with
 const promises = [
   createUser("John", "Doe", "johndoe@email.com", "password"),
   createUser("Jane", "Doe", "janedoe@email.com", "password"),
 ];
 
 // Wait for both promises to resolve
-Promise.all(promises)
-  .then(() => {
-    console.log("Done!");
-  })
-  .catch((err) => {
-    console.log(err);
-  })
-  .finally(() => {
-    db.close();
-    process.exit(0);
-  });
+Promise.all(promises).then(() => {
+  // User account data to seed the database with
+  const promises = [
+    createUserAccount("johndoe@email.com", "instagram", "@johndoe", 0),
```

Comment:
should this be filling the fields in /account/profile as of right now? Because if so, I don't see them on my end:
![image](https://user-images.githubusercontent.com/25871926/199866355-0fb7c4dd-ad9f-4d85-a867-521927217eb3.png)

Reply:
Nope not yet; need to wait for @lelekaz to do this on the frontend.

Reply:
This PR doesn't actually do anything to the front end. It just adds functions to the backend that will be called by the API that the frontend can use.

Reply:
Gotcha, then this is good

Nice work!